app/components/editable-list-element.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { saveEntry } from '../actions/edit-entry';
+import { updateUser } from '../actions/app-data';
+
+const mapDispatch = (dispatch) => {
+    return {
+        saveEntry: (id) => {
+            dispatch(saveEntry(id));
+        },
+        updateUser: (user) => {
+            dispatch(updateUser(user));
+        }
+    };
+};
+
+
+const EditableListElement = ({ saveEntry, user, updateUser, listElement }) => {
+    function updateEntry(event) {
+        const property = event.target.name;
+        const userData = user.data.map(element => {
+            if (element === listElement) {
```

Comment:
Yes this is working since we are having one state and we are getting the data from that.